packages/notifications/src/elements/global-alert/GlobalAlertButton.tsx

```diff
@@ -22,8 +21,15 @@ export const GlobalAlertButton = forwardRef<HTMLButtonElement, IGlobalAlertButto
     return (
       <StyledGlobalAlertButton
         ref={ref}
-        alertType={type}
+        $alertType={type}
         {...props}
+        focusInset={false}
+        isDanger={false}
+        isLink={false}
+        isNeutral={false}
+        isPill={false}
+        isStretched={false}
```

Comment:
Are these `false` props required? Just preventing `Button` prop leaking?

Reply:
That was my interpretation. The values of these props are all `false` by default so we could omit them and expect consumers not to abuse it. I was preserving the props from the underlying view, where these were defined previously.